models/offer.go

```diff
@@ -38,6 +42,37 @@ const enabledOffers = `
 		AND offers.enabled = true
 `
 
+func buildScope(enabledOffers string, filterAttrs map[string]string) string {
+	subQueries := []string{enabledOffers}
+	for k, v := range filterAttrs {
+		// TODO: Possible SQL injection
```

Comment:
That is for the creation of the offer, this part of the code is trying to match the filters against user input.